File: desktop_app/PolyPaint/Modeles/Stats.cs
namespace PolyPaint.Modeles
{
    class Stats
    {
        public int totalGame { get; set; }
        public int bestSoloScore { get; set; }
        public int totalPlayTime { get; set; }
        public string winRate { get; set; }
        public string avgGameTime { get; set; }

        public Stats(int totalGame, int bestScore, int totalPlayTime, string winRate, string avgGameTime) 
        {
            this.totalGame = totalGame;
            this.bestSoloScore = bestScore;
            this.totalPlayTime = totalPlayTime;
            this.winRate = winRate;
            this.avgGameTime = avgGameTime;
        }
    }
}

File: Remise/ClientLeger/app/src/main/java/com/example/client_leger/models/Stats.kt
package com.example.client_leger.models

import android.text.format.DateUtils

class Stats(totalGame: Int, winRate: Double, bestScore: Int, totalPlayTime_s: Int, avgGameTime_s: Double ) {
    var totalGame = totalGame
    var winRate = winRate.toString()
    var bestScore = bestScore
    var totalPlayTime: String = totalPlayTime_s.toString()
    var avgGameTime:String = avgGameTime_s.toString()
}
